.github/workflows/ci.yml

```diff
@@ -67,14 +73,21 @@ jobs:
 
       - name: Install Python packages
         run: |
-          pip install --upgrade codecov tox setuptools
+          pip install --upgrade codecov tox
           pip list
 
       - name: Display tool versions
         run: python misc/build_helpers/show-tool-versions.py
 
-      - name: Run "tox -e py27-coverage"
+      - name: Run coverage tests (Python 2.7)
+        if: matrix.python-version == '2.7'
         run: tox -e py27-coverage
+        timeout-minutes: 20
```

Comment:
Why 20 minutes?  What does this timeout apply to?

Reply:
While working on this PR, a couple of times "tox -e py27-coverage" ran for about 3 hours (or whatever that is GitHub Actions' default timeout), and then failed.  I don't know what causes that behavior, but it is no fun waiting for GitHub Action to finish.  Coverage tests typically finish in about 15 minutes on GA, and if it is going to timeout, might as well do that quicker.